packages/aws-cdk-lib/aws-ecs/lib/container-definition.ts

```diff
@@ -1072,9 +1086,27 @@ export interface PortMapping {
    *
    * For more information, see hostPort.
    * Port mappings that are automatically assigned in this way do not count toward the 100 reserved ports limit of a container instance.
+   *
+   * If you want to expose a port range, you must specify CONTAINER_PORT_UNSET_VALUE as container port.
    */
   readonly containerPort: number;
 
+  /**
+   * The port number range on the container that's bound to the dynamically mapped host port range.
+   *
+   * The following rules apply when you specify a `containerPortRange`:
+   *
+   * - You must use either the `bridge` network mode or the `awsvpc` network mode.
+   * - The container instance must have at least version 1.67.0 of the container agent and at least version 1.67.0-1 of the `ecs-init` package
+   * - You can specify a maximum of 100 port ranges per container.
+   * - A port can only be included in one port mapping per container.
+   * - You cannot specify overlapping port ranges.
+   * - The first port in the range must be less than last port in the range.
+   *
+   * If you want to expose a single port, you must not set a range.
```

Comment:
Also mention they should set `containerPort` to the magic value.

Reply:
I added this requirement as first rule of the list as I think it is neater. Let me know if you prefer to move it to another place